core/domain/config_domain.py

```diff
@@ -147,24 +146,6 @@ class ConfigPropertySchemaDict(TypedDict):
     }
 }
 
-VMID_SHARED_SECRET_KEY_SCHEMA = {
```

Comment:
Please update the comment in line 215 with the deleted config property names.

Reply:
Done